### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by qusai122**
**In this project:**

- Added a desktop version of my portfolio
- added menu detail
- connected menu with the sections
- updated readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +50,10 @@
...
 - **Display Text**
 - **Follow GitHub flow**
-
+- **Display Cards**
+- **Hoover Buttons**
+- **Mobile Version**
+- **Desktop Version**
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
```
Comment (huseyinbiyik) [inline README.md:59]: Please remove Live Demo link and Live Demo heading  if you didn't deploy this project. Currently, the link goes to another project. It is important for having a professional README file.

**Review by huseyinbiyik — COMMENTED** [review]
Hi @qusai122, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request
_Check the comments below_

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (qusai122) [reply README.md:59]: I Deplyed my page and put the right URL.
Thanks for the alert.

**Comment by qusai122** [comment]
**for the next reviewer:** please notice that the live demo is before the merge.

File: style.css [hunk 2]
```diff
@@ -492,3 +521,232 @@ form textarea {
...
+    border-radius: 50%;
+    margin-right: 10px;
+  }
+
+  .sub-skill-name {
+    font-weight: 600;
+    font-size: 15px;
+    color: #253858;
+    padding-top: 0;
+  }
```
Comment (levy002) [inline style.css]: - [x] Kindly make sure the typography of the technology stack matches the Figma design.
- [x] Kindly make sure to add the gap between the text and the image in order to meet the Figma design as much as possible.
  
   - ### YOUR DESIGN
    image

   - ### FIGMA DESIGN
    image

File: style.css [hunk 3]
```diff
@@ -492,3 +521,232 @@ form textarea {
...
+    font-size: 48px;
+    font-weight: 700;
+    padding-left: 6%;
+    border-top-right-radius: 0;
+    width: 40%;
+  }
+
+  .about-me {
+    display: flex;
+  }
```
Comment (levy002) [inline style.css:666]: - [x] Kindly make sure to add enough space between the `Projects` section and the `About me` section in order to meet the Figma design as much as possible.

   - ### YOUR DESIGN
    image

  - ### FIGMA DESIGN
    image

File: style.css [hunk 4]
```diff
@@ -492,3 +521,232 @@ form textarea {
...
+    background-image: url("./assits/desktop-background1.png");
+    background-color: #fff;
+    background-repeat: no-repeat;
+    background-size: 95%;
+    border-radius: 0 0 0 50px;
+  }
+
+  .hero-holder {
+    background-color: #e5e5e5;
+  }
```
Comment (levy002) [inline style.css:653]: - [x] Kindly make sure to add enough space between the headline introduction and the `Work` section in order to meet the Figma design as much as possible.

  - ### Your design
   image

  - ### Figma design
   image

File: style.css [hunk 5]
```diff
@@ -492,3 +521,232 @@ form textarea {
...
+    left: 0;
+    z-index: 1030;
+  }
+
+  .navbar {
+    padding-top: 0;
+    padding-bottom: 0;
+    width: 100%;
+    margin: 0;
+  }
```
Comment (levy002) [inline style.css]: - [x] Kindly make sure the navigation bar is aligned correctly as the one on the Figma design. Please look carefully you will find that the navigation section has the same spacing (padding or margin) on both sides as the same the works section container. It means they are vertically aligned on the same starting point. Kindly make sure to adjust the spacing on both sides of the navigation bar in order to meet the Figma design as much as possible.

   - ### YOUR DESIGN
    image

   - ### FIGMA DESIGN
   
   image

File: style.css [hunk 6]
```diff
@@ -492,3 +521,232 @@ form textarea {
...
+/* Medium devices (landscape tablets, 768px and up) */
+@media only screen and (min-width: 768px) {
+  #menu-icon {
+    display: none;
+  }
+
+  a {
+    text-decoration-line: none;
+    color: #344563;
+  }
```
Comment (levy002) [inline style.css]: - [x] Kindly make sure the color of the links on the navigation bar is the same as the one on the provided Figma design.

  - ### YOUR DESIGN
  
     image

  - ### FIGMA DESIGN
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @qusai122   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional README :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @qusai122 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Comment by qusai122** [comment]
In this project:

- Added a desktop version of my portfolio
- added menu detail
- connected menu with the sections
- updated readme file

**Pull request merged**